extensions/blocks/membership-button/view.js

```diff
@@ -0,0 +1,57 @@
+/* global tb_show */
```

Comment:
Where does this come from? Does `thixbox` need to be a view script dependency?

Reply:
Yes, because it is responsible for displaying an iframe with checkout flow to the actual site visitor